From 55f1a0604bd4df6d591674b5bef123be1af84e55 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Fri, 20 Nov 2020 15:28:49 +0100
Subject: [PATCH] quotacheck: Remove a dead code from process_file()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The ret variable inicialization is useless because it is assigned in
all (two) subsequent code branches.

Signed-off-by: Petr Písař <ppisar@redhat.com>
Signed-off-by: Jan Kara <jack@suse.cz>
---
 quotacheck.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/quotacheck.c b/quotacheck.c
index 5810ced..1b81610 100644
--- a/quotacheck.c
+++ b/quotacheck.c
@@ -672,7 +672,6 @@ Please turn quotas off or use -f to force checking.\n"),
 		}
 	}
 
-	ret = 0;
 	memset(old_info + type, 0, sizeof(old_info[type]));
 	if (is_tree_qfmt(cfmt))
 		ret = v2_buffer_file(qfname, fd, type, cfmt);
-- 
2.26.2

